<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TelnetClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,0,2,0,2,0,0">
       <item>
        <widget class="QLabel" name="labelIP">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditIP">
         <property name="text">
          <string>192.168.1.250</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelUserName">
         <property name="text">
          <string>用户名:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUserName">
         <property name="text">
          <string>huawei</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>密码:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="text">
          <string>Admin@123</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConnect">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDisconnect">
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditSend"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
